apps/examples/testcase/ta_tc/audio/utc/utc_audio_main.c

```diff
@@ -818,13 +823,20 @@ static void utc_audio_pcm_writei_p(void)
 	TC_ASSERT_GEQ_CLEANUP("pcm_writei", fd, 0, clean_all_data(0, buffer));
 
 	printf("Playback start!!\n");
-	do {
+
+	for(;;){
 		num_read = read(fd, buffer, size);
-		if (num_read > 0) {
-			ret = pcm_writei(g_pcm, buffer, pcm_bytes_to_frames(g_pcm, num_read));
-			TC_ASSERT_GEQ_CLEANUP("pcm_writei", ret, 0, clean_all_data(fd, buffer));
+		if (num_read <= 0) {
```

Comment:
TO make this simply, just try writei one more time. please check below pseudo code.
```
ret = pcm_writei();
if (ret == -EPIPE) {
     pcm_prepare();
     ret = pcm_writei();
}
if (ret < 0) {
    TC_ASSERT();
}